src/pytest_html/resources/index.jinja2

```diff
@@ -41,6 +41,7 @@
         </tr>
         <tr class="extras-row">
           <td class="extra" colspan="4">
+            <div class="extraHTML"> </div>
             <div class="media">
               <div class="media-container">
                   <div class="media-container__nav--left"><</div>
```

Comment:
Is this a typo? Double `<`

Reply:
Nope, using the lt app & gt char as arrows